public/app/plugins/datasource/azuremonitor/components/ArgQueryEditor/ArgQueryEditor.tsx

```diff
@@ -57,8 +57,11 @@ const ArgQueryEditor = ({
     datasource
       .getSubscriptions()
       .then((results) => {
+        const selectAllSubscriptionOption = [
+          { label: 'Select all subscriptions', value: 'Select all subscriptions', description: 'Select all' },
```

Comment:
I _think_ ARG may select all subscriptions if the value in the query is `'*'`. Might be worth double checking this

Reply:
If that is a REST query under the hood, then leaving out the subscription parameter in the request will give you all subscriptions:
https://learn.microsoft.com/en-us/rest/api/azureresourcegraph/resourcegraph/resources/resources?view=rest-azureresourcegraph-resourcegraph-2022-10-01&tabs=HTTP

@alyssabull could you please also add the 'managementGroups' parameter?

Reply:
@MaxMelcher we do not currently support management groups. If the parameter were to be specified it'd be empty which I assume is the equivalent of not specifying.

Reply:
@aangelisc - I didn't get that to work and I'm not sure if it will since the query is expecting an array of subscription Ids. Let me know if I missed something